blocks/api/helpers.js

```diff
@@ -0,0 +1,32 @@
+/**
```

Comment:
Function dumping grounds 😩 

As to not be purely cynical: Elsewhere we use "utils" naming? Ought we use that here?